.NET/Microsoft.Recognizers.Definitions.Common/Hindi/DateTimeDefinitions.cs

```diff
@@ -994,30 +992,233 @@ public static class DateTimeDefinitions
             { @"girlsday", new string[] { @"girlsday" } },
             { @"whiteloverday", new string[] { @"whiteloverday" } },
             { @"loverday", new string[] { @"loverday" } },
-            { @"christmas", new string[] { @"christmasday", @"christmas" } },
+            { @"christmas", new string[] { @"christmasday", @"christmas", @"क्रिसमस", @"क्रिसमसकेदिन", @"क्रिसमसकेवालेदिन" } },
             { @"xmas", new string[] { @"xmasday", @"xmas" } },
-            { @"newyear", new string[] { @"newyear" } },
+            { @"newyear", new string[] { @"newyear", @"नयासाल" } },
             { @"newyearday", new string[] { @"newyearday" } },
             { @"newyearsday", new string[] { @"newyearsday" } },
             { @"inaugurationday", new string[] { @"inaugurationday" } },
             { @"groundhougday", new string[] { @"groundhougday" } },
             { @"valentinesday", new string[] { @"valentinesday" } },
-            { @"stpatrickday", new string[] { @"stpatrickday", @"stpatricksday", @"stpatrick" } },
+            { @"stpatrickday", new string[] { @"stpatrickday", @"stpatricksday", @"stpatrick", @"सेंटपैट्रिक्सडे", @"सेंटपैट्रिक्स" } },
             { @"aprilfools", new string[] { @"aprilfools" } },
             { @"stgeorgeday", new string[] { @"stgeorgeday" } },
-            { @"mayday", new string[] { @"mayday", @"intlworkersday", @"internationalworkersday" } },
+            { @"mayday", new string[] { @"mayday", @"intlworkersday", @"internationalworkersday", @"मजदूरदिवस" } },
             { @"cincodemayoday", new string[] { @"cincodemayoday" } },
             { @"baptisteday", new string[] { @"baptisteday" } },
             { @"usindependenceday", new string[] { @"usindependenceday" } },
             { @"independenceday", new string[] { @"independenceday" } },
             { @"bastilleday", new string[] { @"bastilleday" } },
-            { @"halloweenday", new string[] { @"halloweenday", @"halloween" } },
+            { @"halloweenday", new string[] { @"halloweenday", @"halloween", @"हैलोवीन" } },
             { @"allhallowday", new string[] { @"allhallowday" } },
             { @"allsoulsday", new string[] { @"allsoulsday" } },
             { @"guyfawkesday", new string[] { @"guyfawkesday" } },
             { @"veteransday", new string[] { @"veteransday" } },
             { @"christmaseve", new string[] { @"christmaseve" } },
-            { @"newyeareve", new string[] { @"newyearseve", @"newyeareve" } }
+            { @"newyeareve", new string[] { @"newyearseve", @"newyeareve", @"नएसालकीशाम", @"नववर्षकीपूर्वसंध्या", @"न्यूइयरईव" } }
+        };
+      public static readonly Dictionary<int, IEnumerable<int>> HoliDiwaliDates = new Dictionary<int, IEnumerable<int>>
+        {
+            { 1900, new int[] { 3, 16, 10, 23 } },
```

Comment:
Isn't there a way to calculate these instead of adding them all here?
Not blocker.

Reply:
we tried to come up with a formula based on the average duration of the lunar month, but it is difficult to get a reliable result. Using tabulated dates seems a safer solution at the moment. We will update the method in case we find the solution to the problem

Reply:
If it helps, the codebase has some code to calculate lunar holidays. But I have no knowledge of how Diwali is calculated. And it can also vary in different regions, right?

Reply:
ah interesting, will check the codebase! 
Diwali lasts a week and the start day is the same everywhere, but certain states celebrates maybe only one day.